charts/loki/templates/statefulset.yaml

```diff
@@ -131,6 +129,8 @@ spec:
           persistentVolumeClaim:
             claimName: {{ .Values.persistence.existingClaim }}
   {{- else }}
+        - name: tmp
```

Comment:
If .Values.persistence.existingClaim is set, the volume tmp will not be created, so maybe this line can be moved to line 109